.eslintignore

```diff
@@ -11,3 +11,4 @@
 /lib/broccoli/vendor-*.js
 
 !.*
+/node_modules/
```

Comment:
isn't that done by default? how does this relate to the ESLint failure in `.eslintrc.js`?

Reply:
For some reason dotfiles within node_modules are being linted.